### PR TITLE
Add 1.15.3 release

### PR DESCRIPTION
**Pull request by fapaul**

### CODE REVIEW

File: _posts/2022-11-10-release-1.15.3.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+title:  "Apache Flink 1.15.3 Release Announcement"
+date: 2022-11-10T22:00:00.000Z
+categories: news
+authors:
+- danny:
```
Comment (MartijnVisser) [inline _posts/2022-11-10-release-1.15.3.md]: This looks incorrect?

**Review by MartijnVisser — APPROVED** [review]

File: _posts/2022-11-10-release-1.15.3.md (hunk 1 above)
Comment (fapaul) [reply _posts/2022-11-10-release-1.15.3.md]: Yes, indeed, I am gonna fix it. Good catch.

**Review by MartijnVisser — APPROVED** [review]

File: _posts/2022-11-10-release-1.15.3.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+categories: news
+authors:
+- fapaul:
+  name: "Fabian Paul"
+
+excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink 1.15.
+
+---
+
+The Apache Flink Community is pleased to announce the second bug fix release of the Flink 1.15 series.
```
Comment (dannycranmer) [inline _posts/2022-11-10-release-1.15.3.md]:
```suggestion
The Apache Flink Community is pleased to announce the third bug fix release of the Flink 1.15 series.
```

File: _posts/2022-11-10-release-1.15.3.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+## PyPi
+
+* [apache-flink==1.15.3](https://pypi.org/project/apache-flink/1.15.3/)
+
+# Release Notes
+
+<h2>        Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-25554'>FLINK-25554</a>] -         DistributedCacheDfsTest.testSubmittingJobViaRestClusterClient fails on AZP
```
Comment (dannycranmer) [inline _posts/2022-11-10-release-1.15.3.md]: Remove this as per above: "(excluding improvements to the build infrastructure and build stability)"

File: _posts/2022-11-10-release-1.15.3.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29459'>FLINK-29459</a>] -         Sink v2 has bugs in supporting legacy v1 implementations with global committer
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29477'>FLINK-29477</a>] -         ClassCastException when collect primitive array to Python
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29479'>FLINK-29479</a>] -         Support whether using system PythonPath for PyFlink jobs
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29483'>FLINK-29483</a>] -         flink python udf arrow in thread model bug
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29495'>FLINK-29495</a>] -         PulsarSinkE2ECase hang
```
Comment (dannycranmer) [inline _posts/2022-11-10-release-1.15.3.md]: Remove this as per above: "(excluding improvements to the build infrastructure and build stability)"

**Review by dannycranmer — CHANGES_REQUESTED** [review]

File: _posts/2022-11-10-release-1.15.3.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+</dependency>
+```
+
+## Binaries
+
+You can find the binaries on the updated [Downloads page]({{ site.baseurl }}/downloads.html).
+
+## Docker Images
+
+* [library/flink](https://hub.docker.com/_/flink?tab=tags&page=1&name=1.15.3) (official images)
```
Comment (XComp) [inline _posts/2022-11-10-release-1.15.3.md]:
```suggestion
* [library/flink](https://hub.docker.com/_/flink/tags?page=1&name=1.15.3) (official images)
```
Did you copy this from a template or just from the 1.15.2 announcement? We might want to update the template as well if it exist. But I couldn't find anything. :thinking:

File: _posts/2022-11-10-release-1.15.3.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28733'>FLINK-28733</a>] -         jobmanager.sh should support dynamic properties
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28909'>FLINK-28909</a>] -         Add ribbon filter policy option in RocksDBConfiguredOptions
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29134'>FLINK-29134</a>] -         fetch metrics may cause oom(ThreadPool task pile up)
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29158'>FLINK-29158</a>] -         Fix logging in DefaultCompletedCheckpointStore
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29198'>FLINK-29198</a>] -         RetryExtension doesn&#39;t make the test fail if the retries are exhausted
```
Comment (XComp) [inline _posts/2022-11-10-release-1.15.3.md]: This is also more like a build infrastructure issue that was fixed and, therefore, doesn't need to show up in the list.

File: _posts/2022-11-10-release-1.15.3.md (hunk 5 above)
Comment (fapaul) [reply _posts/2022-11-10-release-1.15.3.md]: I do not think there is a template for the web page, but we should think about creating one. I copied mostly the 1.15.2 post that's also causing some of the problems now sorry for that.

**Comment by fapaul** [comment]
Thanks, @XComp and @dannycranmer, for your feedback. PTAL

**Review by dannycranmer — APPROVED** [review]
LGTM, thanks!

File: _posts/2022-11-10-release-1.15.3.md (hunk 5 above)
Comment (XComp) [reply _posts/2022-11-10-release-1.15.3.md]: Ok, I created FLINK-30182 to cover that issue. Thanks for bringing that up @fapaul

**Review by XComp — APPROVED** [review]
LGTM :+1:

**Pull request merged**